nemo/collections/asr/data/audio_to_text_lhotse_prompted.py

```diff
@@ -98,6 +99,7 @@ def __getitem__(self, cuts: CutSet) -> PromptedAudioToTextMiniBatch:
             prompt_lens=prompt_lens,
             prompted_transcript=prompts_with_answers,
             prompted_transcript_lens=prompts_with_answers_lens,
+            cuts=cuts,
```

Comment:
```suggestion
            cuts=cuts.drop_in_memory_data(),
```

I added this method in a new lhotse release to support cases like yours better. What is it doing? `CutSet` can have `Cut`s that either refer to audio data via paths, or that hold audio data in CPU memory. In the latter case, we'd end up transferring potentially lots of large objects across processes (remember this Dataset is placed in a dataloading process and the mini-batch is passed to the training/inference process). With `.drop_in_memory_data()`, this data is replaced by a placeholder so that `CutSet` contains only the metadata.

https://github.com/lhotse-speech/lhotse/blob/master/lhotse/cut/set.py#L1765-L1771

Reply:
ok